### PR TITLE
ignore case in query

### PR DESCRIPTION
**Pull request by maggarwal13**

### CODE REVIEW

File: whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapQueryHandler.java [hunk 1]
```diff
@@ -63,8 +64,7 @@ public void handle(final ResponseObject responseObject) {
                 }
 
                 private void addIfPrimaryObject(final RpslObject responseObject) {
-                    final String primaryKey = responseObject.getKey().toString();
-                    if(primaryKey.equals(query.getSearchValue())) {
+                    if(responseObject.getKey().equals(ciString(query.getSearchValue()))) {
```
Comment (eshryane) [inline whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapQueryHandler.java:67]: You can use responseObject.getKey().equals(query.getSearchValue()) (drop the static ciString)

Comment (maggarwal13) [reply whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapQueryHandler.java:67]: I have to add equals ignore case then, as cistring as our own implementation of equals, wont it be best if we use that?

Comment (eshryane) [reply whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapQueryHandler.java:67]: responseObject.getKey() is already a CIString, and it overrides equals() to perform an insensitive match on a String. So it's unnecessary to also convert the searchValue to a CIString.

Comment (maggarwal13) [reply whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapQueryHandler.java:67]: Ah, I missed the point, Thanks :)

**Pull request merged**